packages/vscode-common/getExtensionApi.ts

```diff
@@ -48,9 +49,6 @@ interface TestHelpers {
 }
 
 export interface CursorlessApi {
-  thatMark: ThatMark;
-  sourceMark: ThatMark;
```

Comment:
note that we no longer expose `that` marks to tests.  They were used for two reasons:

1. Setting `that` mark at beginning of test.  We now do that by exposing a `setThatMark` function in our test helpers, which accepts a plain object rather than a full target, and does rehydration inside the engine rather than letting test harness do it
2. Checking the final that mark.  We now do that by exposing a modified `takeSnapshot` function that already has its own reference to the `ThatMark` objects